### PR TITLE
Fix icon size in contact search popup (way too big)

### PR DESCRIPTION
**Pull request by Jerome-Herbinet**
Signed-off-by: Jérôme Herbinet <33763786+Jerome-Herbinet@users.noreply.github.com>

## Summary

Before : 

After : 

## TODO

- [ ] ...

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by szaimen — CHANGES_REQUESTED** [review]
I fear this will reintroduce https://github.com/nextcloud/server/issues/36832

**Comment by Jerome-Herbinet** [comment]
> I fear this will reintroduce #36832

Why ? What is the relationship between these two problems ?

**Comment by szaimen** [comment]
> Why ? What is the relationship between these two problems ?
> And AFAIK, this other issue has been solved, isn't it ?

See https://github.com/nextcloud/server/pull/36847/files which fixed https://github.com/nextcloud/server/issues/36832

**Comment by Jerome-Herbinet** [comment]
> > Why ? What is the relationship between these two problems ?
> > And AFAIK, this other issue has been solved, isn't it ?
> 
> See https://github.com/nextcloud/server/pull/36847/files which fixed #36832

Sorry @szaimen, but it's not the same problem. I fix an icon's size in the dropdown of contact search in upper right corder or the NC UI. The issue you are mentioning is related to a wrong SVG call in the NC Files right-click menu.

**Comment by szaimen** [comment]
> Sorry @szaimen, but it's not the same problem. I fix an icon's size in the dropdown of contact search in upper right corder or the NC UI. The issue you are mentioning is related to a wrong SVG call in the NC Files right-click menu.

Actually it is related if you look at the PR thst I linked. When merging the proposed change the problem that I linked will get reintroduced because you are basically reverting the PR that I've linked.

**Comment by Jerome-Herbinet** [comment]
It should be better now.

File: core/css/apps.scss [hunk 1]
```diff
@@ -1229,6 +1229,17 @@ $outter-margin: math.div($popoveritem-height - $popovericon-size, 2);
 	}
 }
 
+#contactsmenu .contact .popovermenu {
+	li {
+		> a {
+			> img {
+				width: auto;
+				height: auto;
```
Comment (juliushaertl) [inline core/css/apps.scss:1237]: Still looks off to me: Maybe we just use a fixed width/height of 16px here

Screenshot 2023-04-05 at 16 08 45

**Review by juliusknorr — CHANGES_REQUESTED** [review]

**Review by JuliaKirschenheuter — APPROVED** [review]
Please rebase everything to 1 final commit ;) Thank you!

**Comment by Jerome-Herbinet** [comment]
> Please rebase everything to 1 final commit ;) Thank you!

Done :-)

File: core/css/apps.scss (hunk 1 above)
Comment (Jerome-Herbinet) [reply core/css/apps.scss]: Done with 16px width and height.

**Review by juliusknorr — APPROVED** [review]

**Pull request merged**